pkg/agent/openflow/packetin.go

```diff
@@ -50,16 +50,43 @@ func (c *client) StartPacketInHandler(stopCh <-chan struct{}) {
 	err := c.SubscribePacketIn(uint8(ofprAction), ch)
 	if err != nil {
 		klog.Errorf("Subscribe PacketIn failed %+v", err)
+		return
+	}
+	packetInQueue := workqueue.NewNamed("packetIn")
+	go c.parsePacketIn(packetInQueue, stopCh)
+
+	for {
+		select {
+		case pktIn := <-ch:
+			if packetInQueue.Len() < packetInQueueSize {
```

Comment:
We need a comment here. Especially because if someone refers to the K8s documentation (https://godoc.org/k8s.io/client-go/util/workqueue#Type.Len), they will see this:

> Len returns the current queue length, for informational purposes only. You shouldn't e.g. gate a call to Add() or Get() on Len() being a particular value, that can't be synchronized properly.

Now in our case I think it's ok because we just try to ensure that the queue doesn't grow *too* big, we are not interested in providing an exact guarantee. But a comment would help avoid confusion.

@tnqn

Reply:
Sure. Added comment here.